src/components/Select/index.tsx

```diff
@@ -63,32 +63,39 @@ type StyledSelectProps = Omit<SelectProps, 'errorMesage' | 'hasError' | 'label'>
   label?: string;
 };
 
-const Select: FunctionComponent<SelectProps> = ({
-  errorMessage,
-  hasError,
-  required,
-  theme: propsTheme,
-  label,
-  hintText,
-  ...props
-}) => {
-  const theme = {
-    ...defaultTheme,
-    ...propsTheme,
-  };
+const Select: FunctionComponent<SelectProps> = forwardRef(
+    (
+      {
+        errorMessage,
+        hasError,
+        required,
+        theme: propsTheme,
+        label,
+        hintText,
+        ...props
+      }: SelectProps,
+      ref?: Ref<HTMLInputElement>
+    ) => {
+      
+    const theme = {
+      ...defaultTheme,
+      ...propsTheme,
+    };
 
-  return (
-    <WithLabel label={label} hintText={hintText} required={required} {...props}>
-      <StyledSelect
-        {...props}
-        theme={theme}
-        aria-invalid={hasError || !!errorMessage || false}
-        hasError={hasError || !!errorMessage || false}
-        label={undefined}
-      />
-      {!!errorMessage && <ErrorText>{errorMessage}</ErrorText>}
-    </WithLabel>
-  );
-};
+    return (
+      <WithLabel label={label} hintText={hintText} required={required} {...props}>
+        <StyledSelect
+          {...props}
+          theme={theme}
```

Comment:
you aren't setting a ref prop anywhere here 

you want ref={ref}

Reply:
lol agreed